### ISSUE TITLE
Throw is deprecated use

### ISSUE THREAD

**Issue by sagap** [stylianos-agapiou-sonarsource/sonar-solidity#61]
Use the alternative revert()

**Comment by stylianos-agapiou-sonarsource** [comment]
Issue is raised from the external analyzer ```Solium```.
Rule with the key: security/no-throw
